### PR TITLE
doc: Update documentation for project and template

### LINKED ISSUE #326: Update documentation for project and template
### Summary

Update the information in the description.md and index.md files for the project. And also the contributing.md and index.md files for the template since it is not complete.

### Documentation Page Link

https://osl-incubator.github.io/scicookie/

### Code of Conduct

- [X] I agree to follow the Code of Conduct

### PR DESCRIPTION
**Pull request by YurelyCamacho**
## Pull Request description

Update information in files:

- Project: description.md, index.md, README.md
- Template: index.md, README.md, contributing.md

## How to test these changes

- `...`

## Pull Request checklists

This PR is a:

- [ ] bug-fix
- [ ] new feature
- [ ] maintenance

About this PR:

- [X] it includes tests.
- [ ] the tests are executed on CI.
- [ ] the tests generate log file(s) (path).
- [ ] pre-commit hooks were executed locally.
- [ ] this PR requires a project documentation update.

Author's checklist:

- [ ] I have reviewed the changes and it contains no misspelling.
- [ ] The code is well commented, especially in the parts that contain more
      complexity.
- [ ] New and old tests passed locally.

### CODE REVIEW

File: src/scicookie/{{cookiecutter.project_slug}}/README.md [hunk 1]
```diff
@@ -7,11 +7,246 @@
...
 - Software License: {{ cookiecutter.project_license }}
 {%- endif %}
-- Documentation: {{ cookiecutter.documentation_url }}
+
+- Documentation: https://{{ cookiecutter.project_slug }}.github.io
 
 ## Features
 
-TBD
+{% if cookiecutter.project_layout == "src" %}
```
Comment (xmnlab) [inline src/scicookie/{{cookiecutter.project_slug}}/README.md:15]: features mean features of the project, not the libraries used by the project.

File: src/scicookie/{{cookiecutter.project_slug}}/README.md [hunk 2]
```diff
@@ -7,11 +7,246 @@
 
 - Software License: {{ cookiecutter.project_license }}
 {%- endif %}
-- Documentation: {{ cookiecutter.documentation_url }}
+
+- Documentation: https://{{ cookiecutter.project_slug }}.github.io
```
Comment (xmnlab) [inline src/scicookie/{{cookiecutter.project_slug}}/README.md:11]: we should keep {{ cookiecutter.documentation_url }}

File: src/scicookie/{{cookiecutter.project_slug}}/docs/index.md [hunk 3]
```diff
@@ -11,25 +11,192 @@
 {%- endif %}
 
 ## Features
-{% if cookiecutter.use_bandit == "yes" %}
-- The security of our code: Bandit is a powerful tool that we use in our Python
+
+{% if cookiecutter.project_layout == "src" %}
```
Comment (xmnlab) [inline src/scicookie/{{cookiecutter.project_slug}}/docs/index.md:15]: same here .. features are the features of the project, not the libraries used by the project.

Comment (YurelyCamacho) [reply src/scicookie/{{cookiecutter.project_slug}}/docs/index.md:15]: @xmnlab Perfect, did you check the readme.md and the index.md of the project to see if it is ok?

**Review by xmnlab — APPROVED** [review]
lgtm! thanks @YurelyCamacho

**Pull request merged**